jade-chrome.el

```diff
@@ -35,10 +35,33 @@
 
 (require 'jade-webkit)
 
+(defvar jade-chrome-bin "chromium")
```

Comment:
This should be a `defcustom` instead, so the user can choose between Chrome and Chromium, and pick the binary of this system.